html/assets/css/index.css

```diff
@@ -29,4 +29,8 @@
 .extension-view-project {
 	margin-left: 7px;
 	margin-right: 7px;
+}
+
+.tile {
+	align-items: center;
```

Comment:
Instead of `align-itens` try:

```suggestion
	justify-content: center;
```

Reply:
This changes doesn't effect anything. Weird...

Reply:
There's some weird thing occurring between our browsers. See the fiddle implementation and see if it changes anything.